root.sh

```diff
@@ -127,6 +127,8 @@ cmake $SOURCEDIR
       ${OPENSSL_ROOT:+-DOPENSSL_ROOT=$OPENSSL_ROOT}                                    \
       ${OPENSSL_ROOT:+-DOPENSSL_INCLUDE_DIR=$OPENSSL_ROOT/include}                     \
       ${LIBXML2_ROOT:+-DLIBXML2_ROOT=$LIBXML2_ROOT}                                    \
+      ${LIBXML2_ROOT:+-DPC_LIBXML_INCLUDEDIR=$LIBXML2_ROOT/include}                    \
+      ${LIBXML2_ROOT:+-DPC_LIBXML_LIBDIR=$LIBXML2_ROOT/lib}                            \
```

Comment:
I am surprised one needs to override the package config hooks. Which distribution are you using?

Reply:
I'm doing build inside CentOS-7 and using alibuild --no-system build. This builds CMake 3.19.2 and inside Modules directory is libXML2 expecting the variable PC_LIBXML_LIBDIR (see slc7_x86-64/CMake/v3.19.2-X/share/cmake-3.19/Modules/FindLibXml2.cmake). LibXML2 is not from system but was built using alibuild.